pages/blog/_.vue

```diff
@@ -72,6 +85,23 @@
       },
       heroImage() {
         return this.hero ? this.hero.image.fields.file : null;
+      },
+
+      ...mapGetters({
+        shareUrl: 'http/canonicalUrl',
+        origin: 'http/origin'
+      }),
+
+      identifier() {
+        return `${this.origin}/blog/${this.$route.params.pathMatch}`;
```

Comment:
This should really be delegated to a reusable Vuex getter in the http store, e.g.
```js
  canonicalUrlWithoutLocale(state, getters) {
    return getters.canonicalUrl.replace(/(:\/\/[^/]+)\/[a-z]{2}(\/|$)/, '$1$2');
  }
```